tests/unit-tests/gconstruct/test_converter.py

```diff
@@ -0,0 +1,521 @@
+                },
+                {
+                    "column": "citation_name",
+                    "transformation": {
+                        "name": "huggingface",
+                        "kwargs": {
+                            "action": "tokenize_hf",
+                            "hf_model": "bert",
+                            "max_seq_length": 64
+                        }
+                    },
+                },
+                {
+                    "column": "citation_name",
+                    "transformation": {
+                        "name": "huggingface",
+                        "kwargs": {
+                            "action": "embedding_hf",
+                            "hf_model": "bert",
+                            "max_seq_length": 64
+                        }
+                    },
+                },
+            ],
+            "labels": [
+                {"column": "label", "type": "classification", "split_rate": {"train": 0.8, "val": 0.1, "test": 0.1}}
+            ],
+        }
+    ]
+    gsp_conf["graph"]["edges"] = [
+        {
+            "data": {
+                "format": "parquet",
+                "files": ["/tmp/acm_raw/edges/author_writing_paper.parquet"]
+            },
+            "source": {
+                "column": "~from",
+                "type": "author"
+            },
+            "dest": {
+                "column": "~to",
+                "type": "paper"
+            },
+            "relation": {
+                "type": "writing"
+            },
+            "features": [
+                {"column": "author", "name": "feat"},
+                {
+                    "column": "author",
+                    "name": "hard_negative",
+                    "transformation": {"name": "edge_dst_hard_negative",
+                                       "kwargs": {"separator": ";"}},
+                },
+                {
+                    "column": "num_feature",
+                    "transformation": {"name": "numerical",
+                                       "kwargs": {"normalizer": "standard", "imputer": "mean"}},
+                },
+            ],
+            "labels": [
+                {
+                    "column": "edge_col",
+                    "type": "classification",
+                    "split_rate": {
+                        "train": 0.8,
+                        "val": 0.2,
+                        "test": 0.0
+                    },
+                },
+                {
+                    "column": "edge_col2",
+                    "type": "classification",
+                    "split_rate": {
+                        "train": 0.9,
+                        "val": 0.1,
+                        "test": 0.0
+                    },
+                },
+            ],
+        }
+    ]
+
+    assert len(converter.convert_to_gconstruct(gsp_conf["graph"])["nodes"]) == 1
```

Comment:
Let's assign `converter.convert_to_gconstruct(gsp_conf["graph"])` to a value we re-use, instead of calling the conversion each time.